third_party/proton/proton/profile.py

```diff
@@ -28,17 +46,17 @@ def start(
         ```
 
     Args:
-        name (str): The name (with path) of the profiling session.
-                    If not provided, the default name is "~/proton".
-        backend (str): The backend to use for profiling.
-                       Available options are ["cupti"].
-                       Defaults to "cupti".
-        context (str): The context to use for profiling.
-                       Available options are ["shadow", "python"].
-                       Defaults to "shadow".
-        data (str): The data structure to use for profiling.
-                    Available options are ["tree"].
-                    Defaults to "tree".
+        name (str, optional): The name (with path) of the profiling session.
+                              If not provided, the default name is "~/proton.<profile-format>".
```

Comment:
Explain what `<profile-format>` is?

Reply:
Currently it's just `.hatchet`. Do you prefer me to spell it out?

Reply:
Yeah that seems more clear to the developer.